### PR TITLE
Display the port services listen to from OAP and UI

### PR DESCRIPTION
**Pull request by SheltonZSL**
Based on discussion from https://github.com/apache/skywalking-website/pull/688

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by SheltonZSL** [comment]
I fixed showing irrelevant changes in pull requests (https://github.com/apache/skywalking/pull/12029)

File: changes/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-## Changes are moved to [here](../docs/en/changes/)
+- Update log4j2.xml configurations in both apm-webapp and dist-material.
+
+- Add marker logging in ApplicationStartUp.java within apm-webapp. 
+
+- Change the GRPCServer.java logging in the oap-server/server-library/library-server to marker logging.
+
+- Fix console starts oapService.sh and webappService.sh scripts the startup message is undisplayed. 
+
+- Fix startup.sh starts oapService.sh and webappService.sh on the back-end, the startup message is displayed on the console.
```
Comment (kezhenxu94) [inline changes/README.md:9]: Move these to `docs/en/changes/changes.md`

File: dist-material/log4j2.xml [hunk 2]
```diff
@@ -32,12 +32,22 @@
             </Policies>
             <DefaultRolloverStrategy max="30"/>
         </RollingFile>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout>
+                <pattern>%m%n</pattern>
+            </PatternLayout>
+            <MarkerFilter marker="Server started" onMatch="ACCEPT" onMismatch="DENY"/>
```
Comment (kezhenxu94) [inline dist-material/log4j2.xml:39]: What do you think if we rename the marker to something like `StandardOut` or `Console`?

File: changes/README.md (hunk 1 above)
Comment (SheltonZSL) [reply changes/README.md:9]: OK, I know.

File: dist-material/log4j2.xml (hunk 2 above)
Comment (SheltonZSL) [reply dist-material/log4j2.xml:39]: It should be possible. Let me try.

File: apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java [hunk 3]
```diff
@@ -79,5 +84,9 @@ public static void main(String[] args) throws Exception {
             .build()
             .start()
             .join();
+
+        log.info(startedMarker, "SkyWalking booster UI is now running. " +
+            "OAP services at {} and booster UI at http://localhost:{}",
+            String.join(", ", oapServices), port);
```
Comment (Superskyyy) [inline apm-webapp/src/main/java/org/apache/skywalking/oap/server/webapp/ApplicationStartUp.java:90]: capitalize Booster

File: changes/README.md (hunk 1 above)
Comment (Superskyyy) [reply changes/README.md:9]: And too verbose

**Review by Superskyyy — CHANGES_REQUESTED** [review]
Few minor changes are needed.

File: changes/README.md (hunk 1 above)
Comment (wu-sheng) [reply changes/README.md:9]: Changelogs are featured oriented rather than codes oriented.

**Review by Superskyyy — APPROVED** [review]

**Comment by Superskyyy** [comment]
LGTM

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**